website/docs/fb/flow-launcher.md

```diff
@@ -0,0 +1,57 @@
+---
+id: flow-launcher
+title: Flow Launcher
+---
+
+The Flow Launcher plugin provides a way to launch application via `flow`.
+
+## Dependency
+To use the Flow Launcher, add the following to your `TARGETS`
+```commandline
+//github/facebookresearch/hydra/plugins/hydra_flow_launcher:hydra_flow_launcher
+```
+
+## Usage
+Add hydra/launcher=flow to your command line. Alternatively, override hydra/launcher in your config:
+```commandline
+defaults:
+  - hydra/launcher: flow
+```
+
+<details><summary>Discover Flow Launcher's config</summary>
+
+```commandline title="$ buck run @mode/opt  //path:my_app -- --cfg hydra -p hydra.launcher"
```

Comment:
now we should probably use yaml instead of commandline:
```suggestion
```yaml title="$ buck run @mode/opt  //path:my_app -- --cfg hydra -p hydra.launcher"
```